can/interfaces/ixxat/canlib_vcinpl2.py

```diff
@@ -0,0 +1,920 @@
+        ctypes.c_long,
+        (HANDLE, ctypes.c_uint32, structures.PCANMSG2),
+        __check_status,
+    )
+    # HRESULT canChannelPeekMessage(HANDLE hChannel,PCANMSG pCanMsg );
+    _canlib.map_symbol(
+        "canChannelPeekMessage",
+        ctypes.c_long,
+        (HANDLE, structures.PCANMSG2),
+        __check_status,
+    )
+    # HRESULT canChannelWaitTxEvent (HANDLE hChannel UINT32 dwMsTimeout );
+    _canlib.map_symbol(
+        "canChannelWaitTxEvent",
+        ctypes.c_long,
+        (HANDLE, ctypes.c_uint32),
+        __check_status,
+    )
+    # HRESULT canChannelWaitRxEvent (HANDLE hChannel, UINT32 dwMsTimeout );
+    _canlib.map_symbol(
+        "canChannelWaitRxEvent",
+        ctypes.c_long,
+        (HANDLE, ctypes.c_uint32),
+        __check_status,
+    )
+    # HRESULT canChannelPostMessage (HANDLE hChannel, PCANMSG pCanMsg );
+    _canlib.map_symbol(
+        "canChannelPostMessage",
+        ctypes.c_long,
+        (HANDLE, structures.PCANMSG2),
+        __check_status,
+    )
+    # HRESULT canChannelSendMessage (HANDLE hChannel, UINT32 dwMsTimeout, PCANMSG pCanMsg );
+    _canlib.map_symbol(
+        "canChannelSendMessage",
+        ctypes.c_long,
+        (HANDLE, ctypes.c_uint32, structures.PCANMSG2),
+        __check_status,
+    )
+
+    # EXTERN_C HRESULT VCIAPI canControlOpen( IN  HANDLE  hDevice, IN  UINT32  dwCanNo, OUT PHANDLE phCanCtl );
+    _canlib.map_symbol(
+        "canControlOpen",
+        ctypes.c_long,
+        (HANDLE, ctypes.c_uint32, PHANDLE),
+        __check_status,
+    )
+    # EXTERN_C HRESULT VCIAPI canControlInitialize( IN HANDLE hCanCtl, IN UINT8  bMode, IN UINT8  bBtr0, IN UINT8  bBtr1 );
+    _canlib.map_symbol(
+        "canControlInitialize",
+        ctypes.c_long,
+        (
+            HANDLE,
+            ctypes.c_uint8,
+            ctypes.c_uint8,
+            ctypes.c_uint8,
+            ctypes.c_uint8,
+            ctypes.c_uint32,
+            ctypes.c_uint32,
+            structures.PCANBTP,
+            structures.PCANBTP,
+        ),
+        __check_status,
+    )
+    # EXTERN_C HRESULT VCIAPI canControlClose( IN HANDLE hCanCtl );
+    _canlib.map_symbol("canControlClose", ctypes.c_long, (HANDLE,), __check_status)
+    # EXTERN_C HRESULT VCIAPI canControlReset( IN HANDLE hCanCtl );
+    _canlib.map_symbol("canControlReset", ctypes.c_long, (HANDLE,), __check_status)
+    # EXTERN_C HRESULT VCIAPI canControlStart( IN HANDLE hCanCtl, IN BOOL   fStart );
+    _canlib.map_symbol(
+        "canControlStart", ctypes.c_long, (HANDLE, ctypes.c_long), __check_status
+    )
+    # EXTERN_C HRESULT VCIAPI canControlGetStatus( IN  HANDLE         hCanCtl, OUT PCANLINESTATUS pStatus );
+    _canlib.map_symbol(
+        "canControlGetStatus",
+        ctypes.c_long,
+        (HANDLE, structures.PCANLINESTATUS2),
+        __check_status,
+    )
+    # EXTERN_C HRESULT VCIAPI canControlGetCaps( IN  HANDLE           hCanCtl, OUT PCANCAPABILITIES pCanCaps );
```

Comment:
Maybe consider updating the interface descriptions inline documentation for the changed functions.

Reply:
Done